packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -434,44 +438,64 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
     }, [tileStyle]);
 
     /*
-     * React on changes in the primary color prop.
+     * React on changes in the primary color prop. If not defined, check if it is defined in app config. Otherwise set to default color.
      */
     useEffect(() => {
-        setPrimaryColor(primaryColor);
-    }, [primaryColor]);
+        const defaultPrimaryColor = '#005655'; // --brand-colors-dark-pine-100 from MIDT
+
+        const desiredPrimaryColor = primaryColor ?? appConfig?.appSettings?.primaryColor ?? defaultPrimaryColor;
+        if (desiredPrimaryColor) {
+            setPrimaryColor(desiredPrimaryColor)
+        }
+
+    }, [primaryColor, appConfig]);
 
     /*
-     * React on changes in the start zoom level prop.
+     * React on changes in the start zoom level prop. If not defined, check if it is defined in app config.
      */
     useEffect(() => {
-        setStartZoomLevel(startZoomLevel);
-    }, [startZoomLevel]);
+        const desiredStartZoomLevel = startZoomLevel ?? appConfig?.appSettings?.startZoomLevel ?? null;
+
+        if (!isNullOrUndefined(desiredStartZoomLevel)) {
+            setStartZoomLevel(desiredStartZoomLevel);
+        }
+    }, [startZoomLevel, appConfig]);
 
     /*
-     * React on changes in the pitch prop.
+     * React on changes in the pitch prop. If not defined, check if it is defined in app config.
      * If the pitch is not set, set it to 45 degrees if the view mode switch is visible.
      */
     useEffect(() => {
-        if (!isNullOrUndefined(pitch)) {
-            setPitch(pitch);
-        } else if (viewModeSwitchVisible) {
+        const desiredPitch = pitch ?? appConfig?.appSettings?.pitch ?? null;
+
+        if (!isNullOrUndefined(desiredPitch)) {
+            setPitch(desiredPitch);
+        } else {
             setPitch(45);
```

Comment:
This should only be set to 45 if the 2D/3D switch is visible (viewModeSwitchVisible).

If not, we risk setting the pitch to 45 degrees for solutions that have no 3D features.